LayoutTests/http/tests/site-isolation/resources/getDisplayMedia-starts-frame.html

```diff
@@ -0,0 +1,10 @@
+<script>
+window.internals.settings.setScreenCaptureEnabled(true);
+var streamPromise = null;
+window.internals.withUserGesture(() => {
+	streamPromise = navigator.mediaDevices.getDisplayMedia({ video: true });
```

Comment:
You also need tests to ensure that `getUserMedia({video:true})` and getUserMedia({audio:true})` work in an isolated iframe